spec/unit/resources/splunk_install_spec.rb

```diff
@@ -4,121 +4,121 @@
 describe 'splunk_install' do
   include CernerSplunk::PathHelpers, CernerSplunk::PlatformHelpers
 
-  let(:runner_params) { { platform: 'redhat', version: '7.1' } }
-  let(:test_params) { { name: 'splunk', build: 'cae2458f4aef', version: '6.3.4' } }
+  let(:test_resource) { 'splunk_install' }
+  let(:test_recipe) { 'install_unit_test' }
 
-  let(:windows_opts) { 'LAUNCHSPLUNK=0 INSTALL_SHORTCUT=0 AGREETOLICENSE=Yes' }
+  environment_combinations.each do |platform, version, package, expected_url|
+    describe "on #{platform} #{version}" do
```

Comment:
Similar to my [comment above](https://github.com/cerner/cerner_splunk_ingredient/pull/4/files#r77647371), I think these seem like contexts and not describes since this is describing state.